.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+## Steps to reproduce
+
+How'd you do it?
+
+1. ...
+2. ...
+
+This section should also tell us any relevant information about the
+environment; for example, if an exploit that used to work is failing,
+tell us the victim operating system and service versions.
```

Comment:
I was trying to balance info with brevity here. If it's too long, folks won't read it. Maybe we should move the stuff about logs down around line 20 up here?